tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-e2e-test-nightly-pipeline.yml

```diff
@@ -1,28 +1,19 @@
 trigger: none
 
 jobs:
-- job: Onnxruntime_Linux_GPU_Training_E2E_Test_Nightly
+- job: Orttraining_Linux_GPU_Training_E2E_Test_Nightly
 
   timeoutInMinutes: 120
+  pool: 'Linux-Multi-GPU-V100-E2E2'
```

Comment:
"E2E2"?

Reply:
was wondering about the name. do we need a "2" suffix?

Reply:
was created when there was another pool named Linux-Multi-GPU-V100-E2E